kafka-rest/src/main/java/io/confluent/kafkarest/entities/ConsumerInstanceConfig.java

```diff
@@ -33,22 +37,27 @@
   private EmbeddedFormat format;
   private String autoOffsetReset;
   private String autoCommitEnable;
+  private Integer responseMinBytes;
+  private Integer requestWaitMs;
 
   public ConsumerInstanceConfig() {
     this(DEFAULT_FORMAT);
   }
 
   public ConsumerInstanceConfig(EmbeddedFormat format) {
     // This constructor is only for tests so reparsing the format name is ok
-    this(null, null, format.name(), null, null);
+    this(null, null, format.name(), null, null, null,  null);
   }
 
   public ConsumerInstanceConfig(
       @JsonProperty("id") String id,
       @JsonProperty("name") String name,
       @JsonProperty("format") String format,
       @JsonProperty("auto.offset.reset") String autoOffsetReset,
-      @JsonProperty("auto.commit.enable") String autoCommitEnable
+      @JsonProperty("auto.commit.enable") String autoCommitEnable,
+      @JsonProperty(KafkaRestConfig.PROXY_FETCH_MIN_BYTES_CONFIG)
+              String responseMinBytes,
```

Comment:
It's a little curious that we have to use string types. I thought jackson was pretty clever with type detection.

Reply:
You're most likely right. I'm not familiar with the library